styles/components/_listingsGrid.scss

```diff
@@ -4,6 +4,7 @@
   .listingCard {
     margin: 0 $padMd $padMd 0;
     font-size: $tx5b;
+    border-radius: $corner;
```

Comment:
If the listing card had the content box, we could remove the border-radius here, and remove the border style and width.